test/burn/org/apache/cassandra/net/Verifier.java

```diff
@@ -1326,9 +1339,11 @@ Chunk ensureChunk(long chunkSequenceId) throws InterruptedException
                 Map.Entry<Long, Chunk> e;
                 while ( null != (e = chunkList.firstEntry()) && chunkSequenceId - e.getKey() > 1 << 12)
                 {
+                    if (cancelled)
+                        throw new InterruptedException();
                     WaitQueue.Signal signal = writerWaiting.register();
                     if (null != (e = chunkList.firstEntry()) && chunkSequenceId - e.getKey() > 1 << 12)
-                        signal.await();
+                        signal.await(5L, TimeUnit.SECONDS);
```

Comment:
why is this now timed?

Reply:
This is for safety's sake and in accordance with the following comment which gave me the feeling that we'd get the next chunk within a reasonable timeframe (5 seconds), and if not - we could fail the test, but of course I could be missing something.

https://github.com/apache/cassandra/blob/trunk/test/burn/org/apache/cassandra/net/Verifier.java#L668
```
// TODO: even 2s or 5s are unreasonable periods of time without _any_ movement on a message waiting to arrive
//       this seems to happen regularly on MacOS, but we should confirm this does not happen on Linux
```

Reply:
Sure, ok. Though given the earlier comment might be worth making this timeout a bit bigger to avoid false positives, or perhaps condition the wait time on the system under test.